Makefile

```diff
@@ -654,4 +654,4 @@ golangci-lint:
 	golangci-lint run --timeout 5m
 
 # This endif closes the if at the top of the file
-endif
\ No newline at end of file
+endif
```

Comment:
@guillep2k it seems you ommited the final newline in https://github.com/go-gitea/gitea/commit/23163e98a307300a6ba43e3d8c3b0d031398fef4#diff-b67911656ef5d18c4ae36cb6741b7965R657 which is not confirming [.editorconfig](https://github.com/go-gitea/gitea/blob/6b6f20b6d43b6263320ee872799373f33a751304/.editorconfig#L6). I recommend to install a editorconfig plugin in your editor to automatically fix issues like this.

Reply:
Sorry about that. I didn't realize VSCode required a plugin for that. I've just [installed one](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig). Let's see how it handles ssh/remote. 😄